hazelcast/vector.py

```diff
@@ -0,0 +1,87 @@
+import dataclasses
+import enum
+from typing import Any, Dict, List, Optional, Union
+
+
+__all__ = "Document", "Vector", "IndexConfig", "SearchResult"
+
+
+class Type(enum.IntEnum):
+    DENSE = 0
+
+
+VectorType = Type
```

Comment:
I wonder why we need these variables 🤔 can't we just import Type in other modules for example? (as I see VectorType is used for importing)